File: packages/native-plugins/agent/ios/Sources/AgentPlugin/AgentPlugin.swift
import Capacitor

/// Eliza Agent Plugin — iOS stub.
///
/// On iOS the agent runs on a server and the web implementation
/// handles all communication via HTTP. This native shell is required
/// by Capacitor's plugin system but doesn't need custom native logic.
@objc(AgentPlugin)
public class AgentPlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "AgentPlugin"
    public let jsName = "Agent"
    public let pluginMethods: [CAPPluginMethod] = []
}

File: packages/native-plugins/agent/android/src/main/java/ai/tokagent/plugins/agent/AgentPlugin.kt
package ai.eliza.plugins.agent

import com.getcapacitor.Plugin
import com.getcapacitor.PluginCall
import com.getcapacitor.PluginMethod
import com.getcapacitor.annotation.CapacitorPlugin

/**
 * Eliza Agent Plugin — Android stub.
 *
 * On Android the agent runs on a server and the web implementation
 * handles all communication via HTTP. This native shell is required
 * by Capacitor's plugin system but doesn't need custom native logic.
 */
@CapacitorPlugin(name = "Agent")
class AgentPlugin : Plugin()
